tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -337,8 +337,11 @@ func (c *cluster) compactKV(rev int64) error {
 		_, err = kvc.Compact(ctx, &pb.CompactionRequest{Revision: rev, Physical: true})
 		cancel()
 		conn.Close()
-		if err == nil {
-			return nil
+		if err != nil {
+			if strings.Contains(err.Error(), "required revision has been compacted") && i > 0 {
```

Comment:
@xiang90 if the RPC acks on physical compaction, the Compact needs to be sent to every endpoint to be sure it's physically completed on every node